src/package-linker.js

```diff
@@ -99,11 +99,17 @@ export default class PackageLinker {
     if (pkg.bundleDependencies) {
       for (const depName of pkg.bundleDependencies) {
         const loc = path.join(this.config.generateHardModulePath(ref), this.config.getFolder(pkg), depName);
+        if (await fs.exists(loc)) {
+          try {
+            const dep = await this.config.readManifest(loc, remote.registry);
 
-        const dep = await this.config.readManifest(loc, remote.registry);
-
-        if (dep.bin && Object.keys(dep.bin).length) {
-          deps.push({dep, loc});
+            if (dep.bin && Object.keys(dep.bin).length) {
+              deps.push({dep, loc});
+            }
+          } catch (ex) {
+            // intentionally ignoring error.
```

Comment:
We should probably check that the exception is of a specific type (ENOENT). We still want to throw if we can't read it for any other reason, instead of silencing the error (since it won't get displayed by the missing bundled dependency check either).